components/net/bluetooth_thread.rs

```diff
@@ -128,6 +175,7 @@ impl BluetoothManager {
             cached_services: HashMap::new(),
             cached_characteristics: HashMap::new(),
             cached_descriptors: HashMap::new(),
+            bluetooth_blacklist: BluetoothBlacklist::new(),
```

Comment:
You can't do it here, because it can fail... :( but yeah you should absolutely do it somewhere in the begining